frontend/src/AppRoutes/pageComponents.ts

```diff
@@ -116,11 +116,11 @@ export const MySettings = Loadable(
 );
 
 export const Logs = Loadable(
-	() => import(/* webpackChunkName: "Logs" */ 'pages/LogsModulePage'),
+	() => import(/* webpackChunkName: "Logs" */ 'pages/ModulePage'),
 );
 
 export const LogsExplorer = Loadable(
-	() => import(/* webpackChunkName: "Logs Explorer" */ 'pages/LogsModulePage'),
+	() => import(/* webpackChunkName: "Logs Explorer" */ 'pages/ModulePage'),
```

Comment:
This would require us to test all the flows around the logs explorer change. Can we just consume the views tab by using a separate page?.

Reply:
Done.